spec/ledger_spec.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/ledger.rb'
+
+describe Ledger do
+  subject(:my_ledger) { described_class.new }
```

Comment:
Is it possible for these tests to be isolated from the `Ledger`'s collaborator units?